lib/bbbevents/attendee.rb

```diff
@@ -57,9 +58,16 @@ def csv_row
       ].map(&:to_s)
     end
 
+    def to_hash
+      hash = {}
+      instance_variables.each { |var| hash[var[1..-1]] = instance_variable_get(var) }
+      hash
+    end
+
     def to_json
       hash = {}
       instance_variables.each { |var| hash[var[1..-1]] = instance_variable_get(var) }
+      puts hash.to_json
```

Comment:
A leftover debugging line? Please remove it.

Reply:
Removed.